### PR TITLE
[DNM] change query bad example with one that is more widely used

### PR DESCRIPTION
**Pull request by pxp928**
# Description of the PR

change query bad example with one that is more widely used

# PR Checklist

- [x] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
- [x] All new changes are covered by tests
- [x] If GraphQL schema is changed, `make generate` has been run
- [x] If `collectsub` protobuf has been changed, `make proto` has been run
- [x] All CI checks are passing (tests and formatting)
- [x] All dependent PRs have already been merged

### CODE REVIEW

File: demo/query_bad/query_bad.md [hunk 1]
```diff
@@ -169,18 +167,22 @@ GUAC's external sources (such as Deps.dev). Therefore, if you are seeing less
 than what is shown here, you may want to check back again by re-running the CLI
 in a few minutes!
 
-<img width="1267" alt="6" src="https://user-images.githubusercontent.com/88045217/236021098-184e5bd7-a90d-4509-842c-b6e6988c320d.png">
+<img width="1267" alt="6" src="https://github.com/guacsec/guac/assets/88045217/bd73a5fe-ea6c-46c1-8c56-c52de6c8a567">
```
Comment (lumjjb) [inline demo/query_bad/query_bad.md:170]: kind of curious , does this graph look different with the additional deps.dev data as well?

Comment (pxp928) [reply demo/query_bad/query_bad.md:170]: It does not but we have another demo showing that piece off. I can remove the above snippet that says to check back again if deps.dev is run.

Comment (pxp928) [reply demo/query_bad/query_bad.md:170]: This example shows the effect of various images that need to be updated based on a simple library being compromised.

**Comment by pxp928** [comment]
closing in lieu of https://github.com/guacsec/guac-docs/pull/34

**Pull request closed without merging**